spec/manager/MapFishPrintV2Manager.spec.ts

```diff
@@ -76,23 +85,29 @@ describe('MapFishPrintV2Manager', () => {
         zoom: 2
       })
     });
+    // @ts-ignore
     const manager = new MapFishPrintV2Manager({
       method: 'POST',
       map: map,
       capabilities: mockResponse
     });
-    const getURLMock = 'http://localhost:4321/print/pdf/919288886008494300.png.printout';
+    const getURLMock =
+      'http://localhost:4321/print/pdf/919288886008494300.png.printout';
     manager.init();
 
-    fetch.mockResponse(JSON.stringify({
-      getURL: getURLMock
-    }));
+    // @ts-ignore
+    fetch.mockResponse(
+      JSON.stringify({
+        getURL: getURLMock
+      })
+    );
 
-    return manager.print()
-      .then(payload => {
-        expect(payload).toEqual(getURLMock);
-        fetch.resetMocks();
-      });
+    // @ts-ignore
+    return manager.print().then(payload => {
+      expect(payload).toEqual(getURLMock);
+      // @ts-ignore
```

Comment:
Any chance to get rid of all the `@ts-ignore`s?

Reply:
I think this would require quite a lot of rewriting. The types of a lot of return values etc. are not properly detected and requires a lot of type annotations to work properly, mainly due to class inheritance regarding the base manager class.